redash/query_runner/pg.py

```diff
@@ -182,6 +184,8 @@ def run_query(self, query, user):
 class Redshift(PostgreSQL):
     default_doc_url = ("http://docs.aws.amazon.com/redshift/latest/"
                        "dg/cm_chap_SQLCommandRef.html")
+    data_source_version_query = "select version()"
+    data_source_version_post_process = "split by space take last"
```

Comment:
Oh the implementation of this leaves MUCH to be desired, but given that as this version getting feature is implemented in other query_runners there will only be more variations on this theme I thought it was important to keep the logic human readable as opposed to in numbers. Also the `else` here will make it easy to spot if there is a typo in these.